layouts/swagger/baseof.html

```diff
@@ -22,8 +22,8 @@
           </aside>
           <main class="col-12 col-md-9 col-xl-8 ps-md-5" role="main">
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
-            <script src="{{ "js/swagger-ui-bundle.js" | relURL }}"></script>
-            <script src="{{ "js/swagger-ui-standalone-preset.js" | relURL }}"></script>
+            <script src="https://unpkg.com/swagger-ui-dist@5.1.0/swagger-ui-bundle.js" crossorigin></script>
```

Comment:
I would propose to add the SRI hash here (as we do with other scripts).
Same for other files pulled in from `unpkg.com`.

Reply:
That works for me.

Reply:
Edited to add the SRI hash. Thanks!

TIL that you can append `?meta` to any package in unpkg to retrieve package metadata, including SRI hashes: https://github.com/mjackson/unpkg/issues/48#issuecomment-323161338. That, or using srihash.org.

Reply:
> Edited to add the SRI hash. Thanks!
> 
> TIL that you can append `?meta` to any package in unpkg to retrieve package metadata, including SRI hashes: [mjackson/unpkg#48 (comment)](https://github.com/mjackson/unpkg/issues/48#issuecomment-323161338). That, or using srihash.org.

Thanks for the info. Up to now, I didn't know that either.